packages/eslint-config-fbjs-opensource/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-fbjs-opensource",
-  "version": "2.0.1",
+  "version": "3.0.0",
```

Comment:
Personally I prefer specific commits that bump versions, not in PRs (so we can land safely then release)